docs/concepts/Auditing-Packages.md

```diff
@@ -125,6 +125,15 @@ If security vulnerabilities are found and updates are available for the package,
 - Use the NuGet package manager user interface in Visual Studio to update the individual package.
 - Run the `dotnet add package` command with the respective package ID to update to the latest version.
 
+#### Transitive Packages
+
+If a known vulnerability exists in a top-level package's transitive dependencies, you have these options:
+
+- Add the fixed package version as a direct package reference. **Note:** Be sure to remove this reference when a new package version update becomes available.
+- Use [Central Package Management with the transitive pinning functionality](https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management#transitive-pinning).
+- [Suppress the advisory](https://learn.microsoft.com/nuget/concepts/auditing-packages#excluding-advisories) until a package version update becomes available.
+- File an issue in the top-level package's tracker to request an update.
```

Comment:
Neither of these should be recommended as alternatives to the first.

A suppression should only be used if for some reason the project owner cannot update a dependency.  For example - they aren't responsible for deploying the dependency and need to keep the version pinned to the version some host process is responsible for deploying.

Blocking on the external package to update is not recommended either.  They should inform the package owner that they'd like an update, but they shouldn't block on it.

Reply:
hopefully these read in the sense that they aren't mutually exclusive. just options people have and should consider.

some of these can be combined and enough options exist to not block.

Reply:
I feel strongly that we cannot suggest this:
> [Suppress the advisory](https://learn.microsoft.com/nuget/concepts/auditing-packages#excluding-advisories) until a package version update becomes available.

That implies that the user is blocked and should suppress.  That's not the case.

Reply:
ill work to rephrase, the idea is this:

a developer sees an advisory, they look, they don't assess they are affected, they suppress it, and next time they update packages they get a non-vulnerable one and can remove the suppression.

similar to the promotion of top-level dependency flow.

this helps them unblock in the case they may have warningsaserrors and need to make a choice of promotion or suppression.

Reply:
Rather than say `until a package version update becomes available.` you could say `until it can be addressed.`  This means that addressing it might happen in one of many ways